.github/workflows/build_tests.yml

```
@@ -0,0 +1,26 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  Build Tests:
```

actionlint (1.7.4)

invalid job ID "Build Tests". job ID must start with a letter or _ and contain only alphanumeric characters, -, or _

(id)